reactive/spark.py

```diff
@@ -123,5 +119,3 @@ def client_present(client):
 @when_not('spark.started')
 def client_should_stop(client):
     client.clear_spark_started()
-
-
```

Comment:
There have been some ongoing discussions about testing reactive charms, but we haven't had the time to make progress on them.  One idea I had is to have a "dry-run" mode in reactive such that you could have a unit test that manually sets states then calls reactive in dry-run mode, and it would tell you which handlers would be triggered without actually triggering them.  I'm not certain if that sort of test would add enough value for the amount of coding it would require, but it would at least let you vet your `@when` / `@when_not` decorators.  But probably integration tests are a more reliable way to verify that.